### DISCUSSION TITLE
What data should we collect?

### DISCUSSION THREAD

**Discussion by thisisnic** [thisisnic/shinyarrowpi#1] [Ideas]
What data should we collect?

**Comment by thisisnic** [comment]
https://shop.pimoroni.com/products/enviro?variant=31155658457171 looks like it could be cool - collect environment data

**Reply by thisisnic** [reply]
See also this: https://www.cbc.ca/news/science/gas-stoves-air-pollution-1.6394514

**Comment by sellorm** [comment]
The enviro+ can report the following data:

* Sensor: example reading
* temperature: 19.0 C
* air pressure: 1028.5 hPa
* humidity: 20.7 %
* light: 0.0 Lux
* oxidising gases: 1.0 kO
* reducing gases: 181.8 kO
* nh3 (amonia): 1.8 kO

"oxidising" and "reducing" are gas resistance values, which I confess, I don't fully understand at present. We may be able to pull out the specific gas readings from the sensor with a little more digging.

Apparently the sensor supports:

* Carbon monoxide (CO)
* Nitrogen dioxide (NO2)
* Ethanol (C2H5OH)
* Hydrogen (H2)
* Ammonia (NH3)
* Methane (CH4)
* Propane (C3H8)
* Iso-butane (C4H10)

**Reply by sellorm** [reply]
From the enviro+ docs...

> The MICS6814 is an analog gas sensor that can detect three different groups of gases that they refer to in the datasheet as reducing, oxidising, and NH3. The major gases/vapours that the sensor detects are: carbon monoxide (reducing), nitrogen dioxide (oxidising), and ammonia (NH3), but it is also sensitive to others, including hydrogen, ethanol, and hydrocarbons.
>
> Each of the three groups of gases is effectively its own sensor within the MICS68141, and the analog voltage readings that the sensor produces are read by an analog to digital converter (ADC) and then converted into resistances by our code. These resistances will range from low hundreds of Ohms to tens of thousands of Ohms, and vary depending on the levels of each group of gases.
>
> Because each group of gases could be a mix of different gases, it's not possible to single out any one gas specifically or to quantify their levels precisely, so the best way to interpret the data is to take readings until they stabilise, set a baseline, and then look for changes relative to that baseline. This gives you a rough idea of whether the air quality is increasing or decreasing.
>
> The reducing and NH3 resistance readings will drop with increasing concentrations of the gases that they detect, and the oxidising sensor will increase with increasing levels of nitrogen dioxide.

**Reply by sellorm** [reply]
So we are limited to those three gas readings, but at least we can interpret/represent them more accurately.
